### PR TITLE
Introduce pages components structure

### PR DESCRIPTION
**Pull request by fe51**
The aim of this PR is to structure the project with pages components as discussed yesterday

Also, I have been testing a stepper for the add reporting process (from [Material UI](https://mui.com/material-ui/react-stepper/))

It introduces the following modifications : 
- Adds pages folder with one component file per page
- Updates App by adding newpages : report-an-add and about-greenwashing
- Adds a mobile stepper
- Updates import order (build in, external, internal)
- Updated packages accordingly

happy to discuss it and feedbacks are welcome

### CODE REVIEW

File: src/App.tsx
Comment (alxthm) [inline src/App.tsx]: typo: should be ReportAnAd ? and report-ad, in the file names, URL, etc

File: src/pages/about-greenwashing.tsx
Comment (alxthm) [inline src/pages/about-greenwashing.tsx:3]: ~~h1 instead if it's the first header? ([better for accessibility](https://www.a11yproject.com/posts/how-to-accessible-heading-structure/))~~ my bad, there is already a h1 header above

**Review by alxthm — COMMENTED** [review]
great work 🎉

File: src/App.tsx
Comment (fe51) [reply src/App.tsx]: Hehe nice catch ! 
As long as I do not have computer at hand,
Either you can push this typo correction 
Or I merge the PR and you can provide a quick fix later ! 
Tell me :)

Comment (alxthm) [reply src/App.tsx]: done!

File: src/pages/report-ad.tsx
Comment (alxthm) [inline src/pages/report-ad.tsx:122]: I changed `==` to `===`, since [it's generally safer in JS](https://stackoverflow.com/questions/359494/which-equals-operator-vs-should-be-used-in-javascript-comparisons)?

Comment (fe51) [reply src/pages/report-ad.tsx:122]: Perfect ! Being a JS noob I was not sure, you are right it will be safer !

**Comment by fe51** [comment]
Thanks for the edit ! I will merge once one of you approved the PR :)

**Review by alxthm — APPROVED** [review]
lgtm 👍

**Pull request merged**
